ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -514,3 +528,13 @@ def get_gae(rewards, value_estimates, value_next=0.0, gamma=0.99, lambd=0.95):
     delta_t = rewards + gamma * value_estimates[1:] - value_estimates[:-1]
     advantage = discount_rewards(r=delta_t, gamma=gamma * lambd)
     return advantage
+
+
+def get_devices(multi_gpu):
```

Comment:
This seems like a utility function.  I'd probably put it in some "tensorflow util" module or put it with the multi gpu model file.

Reply:
created a new multi gpu model file and moved this there